ikarus/finiteelements/mechanics/materials/materials.cpp

```diff
@@ -9,11 +9,22 @@
 #include <ikarus/finiteelements/mechanics/materials/linearelasticity.hh>
 #include <ikarus/finiteelements/mechanics/materials/neohooke.hh>
 #include <ikarus/finiteelements/mechanics/materials/svk.hh>
+#include <ikarus/finiteelements/mechanics/materials/vanishingstrain.hh>
+#include <ikarus/finiteelements/mechanics/materials/vanishingstress.hh>
 
 namespace Ikarus {
 
 template struct LinearElasticityT<double>;
 template struct NeoHookeT<double>;
 template struct StVenantKirchhoffT<double>;
 
+// // Forward compile planeStress and planeStrain
+// template auto planeStress(const LinearElasticity& mat, typename LinearElasticity::ScalarType tol);
```

Comment:
What are these? Do you want to include them or not?

Reply:
I am actually not sure if this helps with compile times, do you think its beneficial to compile the planestress and planestrain before hand or if it even works like that?

Reply:
So I think i would remove it and only precompile the materials for now.
In Python there is now additionaly compilation anyway due to the `asPlaneStress/Strain` mechanism.